package.json

```diff
@@ -21,17 +21,19 @@
     "axios": "^0.21.1",
     "cross-fetch": "^3.1.4",
     "framer-motion": "^3.2.1",
-    "next": "^10.0.4",
+    "next": "^12.0.7",
```

Comment:
Woah, wait, why is your PR updating nextjs version?

Reply:
I have no idea. Don't even know if it's bad or good.

Reply:
❌  Let's not be doing major upgrades like this. This is dangerous.

Good catch @swarajpure